dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -214,6 +218,14 @@
 		status = "disabled";
 		label = "OS_TIMER";
 	};
+
+	trng: random@50138000 {
+		compatible = "nxp,kinetis-trng";
+		reg = <0x50138000 0x4000>;
```

Comment:
Is the base address correct? I think it should be the offset from 0x5000_0000, which would make it 0x138000

Reply:
Why do you think it should be this number? 

I pulled the address from TRNG_BASE in the MIMXRT685S_cm33.h file. I'm actually unsure if it should be 50138000 or 4013800... secure vs non-secure address space

Reply:
okay... now I understand after discussing this with Kumar. Changes made.